### ISSUE TITLE
npm file size pretty big

### ISSUE THREAD

**Issue by DenisCarriere** [nashwaan/xml-js#10]
Just noticed that your published npm bundle is pretty large, almost 1MB (824KB).

```bash
$ du -h node_modules/xml-js
```
```bash
184K    xml-js/artwork
 12K    xml-js/bin
 32K    xml-js/lib
156K    xml-js/test/browse-jasmine/lib/jasmine-2.5.2
156K    xml-js/test/browse-jasmine/lib
452K    xml-js/test/browse-jasmine
520K    xml-js/test
824K    xml-js
```
```bash
$ ls -lh
```

```bash
-rw-r--r--   1 mac  staff   1.1K May  2  2016 LICENSE
-rw-r--r--   1 mac  staff    17K Nov  4 00:15 README.md
-rw-r--r--   1 mac  staff   456B Jun  7  2016 appveyor.yml
drwxr-xr-x   5 mac  staff   170B Jan 24 10:07 artwork
drwxr-xr-x   4 mac  staff   136B Jan 24 10:07 bin
-rw-r--r--   1 mac  staff    34B Sep  7 03:42 index.js
drwxr-xr-x   8 mac  staff   272B Jan 24 10:07 lib
-rw-r--r--   1 mac  staff   5.2K Jan 24 10:07 package.json
drwxr-xr-x  13 mac  staff   442B Jan 24 10:07 test
```

Your core javascript files (`index.js` + `lib/`) is only 32KB.

Inside your test cases you've got 520KB that you ship with your npm bundle also `artwork` is 180KB, might be worth adding an `.npmignore` file to your repo and exclude some of the files that are not needed.

**Proposed `.npmignore`

```
artwork/
test/
```

**Comment by DenisCarriere** [comment]
Or you can include `files` in your `package.json`.

Example
```json
  "files": [
    "index.js",
    "index.d.ts"
  ],
```

**Comment by nashwaan** [comment]
@DenisCarriere  Thanks for this great tip. Your pull request has been merged.

**Comment by DenisCarriere** [comment]
Awesome stuff! Let me know when you publish a patch release, I'll update my dependencies

**Comment by nashwaan** [comment]
v1.01 is published to address this issue.

Thanks again for your suggestion. I wouldn't have realized it by myself.

**Comment by DenisCarriere** [comment]
@nashwaan Thanks,

However, now you've got new files 😢 `_playground` which is 364K.

```bash
$ du node_modules/xml-js -h
4.0K	./_playground/node-xml2json-master/bin
 20K	./_playground/node-xml2json-master/lib
104K	./_playground/node-xml2json-master/test/fixtures
132K	./_playground/node-xml2json-master/test
184K	./_playground/node-xml2json-master
 44K	./_playground/sax-js-master/lib
 76K	./_playground/sax-js-master
364K	./_playground
 12K	./bin
 32K	./lib
 16K	./types
640K	.
```

It's kinda tricky when you publish with npm, it literally takes your entire folder and publishes that. I forgot when you add `.npmignore` it stops following the `.gitignore` rules.

**Comment by nashwaan** [comment]
Pull request has been merged. Published in v1.0.2.
Hope it is ok now.

**Comment by DenisCarriere** [comment]
@nashwaan Yay! that works, it's a lot smaller now 👍 

**Now v1.0.2**
```bash
 44K	./sax/lib
 76K	./sax
 12K	./xml-js/bin
 32K	./xml-js/lib
 16K	./xml-js/types
120K	./xml-js
200K	.
```

**Before v0.9.6**
```bash
4.0K	./.bin
 44K	./sax/lib
 76K	./sax
184K	./xml-js/artwork
 12K	./xml-js/bin
 32K	./xml-js/lib
156K	./xml-js/test/browse-jasmine/lib/jasmine-2.5.2
156K	./xml-js/test/browse-jasmine/lib
452K	./xml-js/test/browse-jasmine
516K	./xml-js/test
820K	./xml-js
900K	.
```

I'd say that's a 450% filesize reduction.
